src/NeonView.js

```diff
@@ -158,10 +158,15 @@ function NeonView (params) {
                 }
             }
         });
-        $("#svg_output").on("touchstart", (evt) => {
-            console.log("Touches: " + evt.touches.length);
-            if (evt.touches.length === 2) {
```

Comment:
It may be wise to try a cleaner git history approach, if you are going to delete most of the previous commit. This is just a thought.